src/mmw/js/src/core/chart.js

```diff
@@ -46,7 +46,14 @@ function handleCommonOptions(chart, options) {
         chart.yAxis.axisLabel(options.yAxisLabel);
     }
     if (options.isPercentage) {
-        chart.yAxis.tickFormat(d3.format('.0%'));
+        // If the max proportion area is less than 1%, include
+        // 2 digits of precision on the y axis label, otherwise
+        // use whole integers
+        if (options.maxYValue < .01) {
```

Comment:
This `if` statement could be further graduated to display a more granular precision based on max Y values (...`< .001`, etc), but I thought this was a reasonable middle ground.